adapters/ix/ix_test.go

```diff
@@ -276,3 +294,220 @@ func TestMakeRequestsErrIxDiag(t *testing.T) {
 	_, errs := bidder.MakeRequests(req, nil)
 	assert.Len(t, errs, 1)
 }
+
+func TestSetFeatureToggles(t *testing.T) {
```

Comment:
It's fine if you want to add unit tests, but we request code coverage via the json framework as that performs additional checks such as for memory corruption issues. We only consider `TestJsonSamples` when evaluating code coverage.